src/applications/check-in/hooks/useSessionStorage.jsx

```diff
@@ -46,6 +46,20 @@ const useSessionStorage = (isPreCheckIn = true, maxValidateAttempts = 3) => {
     [SESSION_STORAGE_KEYS],
   );
 
+  const setComplete = useCallback(
+    (window, complete) => {
+      setSessionKey(window, SESSION_STORAGE_KEYS.COMPLETE, { complete });
```

Comment:
This is now the second (at least) piece of data we are storing in session, should we start to think of session structuce?

Reply:
let's chat about it after ToT today if you're free :-)